xml/regression.sh

```diff
@@ -62,11 +67,11 @@ log2_or_0_add() {
 	_lo0a_op1=${1:?}
 	_lo0a_op2=${2:?}
 
-	if test ${_lo0a_op1} -gt ${_lo0a_op2}; then
+	if [ "$_lo0a_op1" -gt "$_lo0a_op2" ]; then
```

Comment:
Numeric operations like -gt shouldn't use quotes around the values. If user input is involved, you can validate the values are numeric beforehand, but if they're determined internally there's no need.

Reply:
Why not? The comparison is still numeric even if double-quoted. Double-quoting is just a best-practice to ensure the interpolated variable is treated as a single token regardless of whitespace.

The examples on TLDP's bash guide use double quotes: https://tldp.org/LDP/abs/html/comparison-ops.html

Reply:
I don't remember where I saw that recommendation, and it apparently doesn't matter, so no big deal. I do find `"$#"` looks weird since it's a built-in numeric variable.